src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -41,12 +61,16 @@ const ButtonGroup = ({ children, disabled, ...props }) => (
 ButtonGroup.propTypes = {
    /** Buttons to be grouped together. */
    children: arrayOf(node).isRequired,
+   /** Indicated whether button group should display in as a row or
+    * stacked in a column */
+   direction: string,
```

Comment:
What do you think about making this perhaps be a bool: `horizontal: bool` since there are only two possible values? Also, 'row' isn't really a direction.

Reply:
direction: 'row' seems to make sense in this case though, since `flex-direction` either takes `row` or `column`

Reply:
Yeah, I actually originally implemented this with a `vertical: bool`, so essentially the same idea, but I changed it to 'direction' because I thought it more closely mirrored what it is actually doing, which is setting the flex-direction. I think re-naming it to `flexDirection` is a little too specific. You don't necessarily want to know the implementation details from the prop names.